lib/model.js

```diff
@@ -1617,9 +1595,8 @@ Model.prototype.aggregate = function(attribute, aggregateFunction, options) {
  * @return {Promise<Integer>}
  */
 Model.prototype.count = function(options) {
-  options = Utils._.clone(options || {});
-
-  _.defaults(options, { hooks: true });
+  options = Utils.cloneDeep(options);
+  options.hooks = true;
```

Comment:
This line needs to remain `_.defaults(options, { hooks: true });` to keep same behaviour. `hooks: true` is a default, not compulsory.